hathor/simulator/simulator.py

```diff
@@ -250,7 +256,13 @@ def run_until_complete(self,
     def run(self,
             interval: float,
             step: float = DEFAULT_STEP_INTERVAL,
-            status_interval: float = DEFAULT_STATUS_INTERVAL) -> None:
+            status_interval: float = DEFAULT_STATUS_INTERVAL,
+            *,
+            trigger: Optional[Trigger] = None) -> bool:
         assert self._started
         for _ in self._run(interval, step, status_interval):
-            pass
+            if trigger is not None and trigger.should_stop():
+                return True
+        if trigger is not None:
+            return False
+        return True
```

Comment:
What does `True` mean in this case?

Reply:
I'll add a docstring for this method.

Reply:
Done! What do you think?

Reply:
By reading the code, I had understood that `True` meant "stopped by trigger". With the docstring it makes sense, thanks!